manifests/init.pp

```diff
@@ -151,6 +151,7 @@
   $alerts                     = $::prometheus::params::alerts,
   Array $alert_relabel_config = $::prometheus::params::alert_relabel_config,
   Array $alertmanagers_config = $::prometheus::params::alertmanagers_config,
+  String $retention           = $::prometheus::params::storage_retention,
```

Comment:
Datatypes \o/

Reply:
is this a request for a more specific datatype or just celebrating the current `String` datatype? :smile: 
I mean, I _could_ use a regexp type, but it felt a bit overkill...